From 0a347c1876497eb740ead3490b9fb539aa1e6d67 Mon Sep 17 00:00:00 2001
From: Bolshov Maxim <>
Date: Sat, 27 Dec 2025 20:36:45 +0300
Subject: [PATCH] Update CMake minimum versions to 3.10

Update CMake minimum required versions to 3.10 to suppress
deprecation warnings when building with newer CMake:
- Update main CMakeLists.txt from 3.1 to 3.10
- Update zycore/CMakeLists.txt from 3.9 to 3.10
---
 CMakeLists.txt        | 2 +-
 zycore/CMakeLists.txt | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 709d827..6bf3471 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
 
 project(Zydis VERSION 3.2.1.0 LANGUAGES C CXX)
 
diff --git a/zycore/CMakeLists.txt b/zycore/CMakeLists.txt
index 26e0882..8ac4e68 100644
--- a/zycore/CMakeLists.txt
+++ b/zycore/CMakeLists.txt
@@ -2,7 +2,7 @@ if (TARGET Zycore)
     return()
 endif ()
 
-cmake_minimum_required(VERSION 3.9 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
 
 project(Zycore VERSION 1.1.0.0 LANGUAGES C CXX)
 
-- 
2.51.0

